books/management/commands/resync.py

```diff
@@ -71,7 +71,7 @@ def handle(self, *args, **options):
             if book:
                 if success:
                     counter['success'] = counter['success'] + 1
-                    self.stdout.write(self.style.SUCCESS(
+                    self.stdout.write(self.style.HTTP_REDIRECT(
```

Comment:
This was mainly due to the downgrading of Django to 1.8, which does not have the `SUCCESS` decorator. The seemingly weird `HTTP_REDIRECT` style is basically for having the successfully modified/imported epubs to be shown in a light green, mainly to be able to check at a glance for problems (the idea was to leave the red for errors).

If you feel it makes sense to have both the successful and the failed ones in red or another combination of colors, just go for it, no worries!